pandas/core/resample.py

```diff
@@ -1519,8 +1519,7 @@ def __init__(
         self.label = label
         self.kind = kind
 
-        self.convention = convention or "E"
-        self.convention = self.convention.lower()
```

Comment:
will this mess up if the user passes "E"?

Reply:
See line 1488 above, we already have restricted the values a lot.

Also, I think probably the whitespace should get removed around this block since convention is now behaving just like kind, how, etc

Reply:
thanks